ios/SundarGutka/LaunchScreen.storyboard

```diff
@@ -1,91 +1,38 @@
-                                <rect key="frame" x="180.66666666666666" y="105.66666666666667" width="66.666666666666657" height="39.666666666666671"/>
-                                <fontDescription key="fontDescription" name="GurbaniAkharThickSG" family="GurbaniAkharThickSG" pointSize="30"/>
-                                <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" verticalCompressionResistancePriority="751" text="pRkwSk:" textAlignment="center" lineBreakMode="middleTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="nhk-LM-aeN">
-                                <rect key="frame" x="0.0" y="807" width="428" height="25.666666666666629"/>
-                                <fontDescription key="fontDescription" name="GurbaniAkharSG" family="GurbaniAkharSG" pointSize="19"/>
-                                <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="khalis-logo" translatesAutoresizingMaskIntoConstraints="NO" id="uif-bk-E4u">
-                                <rect key="frame" x="122.66666666666667" y="832.66666666666663" width="182.66666666666663" height="59.333333333333371"/>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="SplashScreen" translatesAutoresizingMaskIntoConstraints="NO" id="083-jd-HN7">
+                                <rect key="frame" x="312" y="283" width="400" height="800"/>
                                 <constraints>
-                                    <constraint firstAttribute="width" secondItem="uif-bk-E4u" secondAttribute="height" multiplier="40:13" id="2Nc-2e-SMa"/>
+                                    <constraint firstAttribute="width" secondItem="083-jd-HN7" secondAttribute="height" multiplier="1:2" id="ObO-Gu-qLK"/>
+                                    <constraint firstAttribute="height" constant="800" id="Wkv-Nj-jh9"/>
+                                    <constraint firstAttribute="width" constant="400" id="lXN-o7-euU"/>
                                 </constraints>
+                                <string key="userLabel" base64-UTF8="YES">
+CQ
+</string>
```

Comment:
Fix indentation here. I would recommend you to use linter in your project to avoid these indentation issues.

Reply:
Fix @inderpreetsingh  Veer jio.